code/datums/station_traits/neutral_traits.dm

```diff
@@ -189,3 +189,124 @@
 		return
 
 	gorilla.poll_for_gorilla()
+
+/datum/station_trait/birthday
+	name = "Employee Birthday"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 2
+	show_in_report = TRUE
+	report_message = "We here at Nanotrasen would all like to wish Employee Name a very happy birthday"
+	trait_to_give = STATION_TRAIT_BIRTHDAY
+	blacklist = list(/datum/station_trait/announcement_intern, /datum/station_trait/announcement_medbot) //Overiding the annoucer hides the birthday person in the annoucement message.
+	var/mob/living/carbon/human/birthday_person
```

Comment:
Minor nitpick and it's a pretty self-explanatory var anyway but this is lacking a documenting comment